Sources/GDSCommon/UI/UIKit/SharedViews/Patterns/Scanner/ScanningViewController.swift

```diff
@@ -79,6 +79,7 @@ public final class ScanningViewController<CaptureSession: GDSCommon.CaptureSessi
         makeScannerCaptureView()
         updateRegionOfInterest()
         addImageOverlay()
+        setBackButtonTitle()
```

Comment:
Should this be here?

Reply:
Was hoping to include it In this PR but since it is a separate issue I will remove